index.html

```diff
@@ -180,14 +180,15 @@
         </li>
         <li>When the `type` attribute of the [link] element of a <a>preload
         link</a> that is already [in a Document] but was previously not
-        obtained due to the `type` attribute specifying an unsupported
+        obtained due to the `type` attribute not specifying a
+        [parsable MIME type] or specifying an [unsupported MIME type] for the
         <a>request destination</a> is set, removed, or changed.
```

Comment:
Not really.
Implementation wise, MIME type checks happen in the context of the resource type (e.g. "is 'font/woff2' supported for images?" will return false).

I'm not sure how to hook it up. Ideas would be appreciated.